ansible/files/telemetry.sh

```diff
@@ -67,6 +67,24 @@ sudo sh -c "sudo cat <<EOF > /etc/presto/conf/jvm.config
 EOF"
 sudo pkill presto
 
+# upgrade hive
+tmp="$(mktemp -d)"
+cd "$tmp"
+aws s3 sync s3://telemetry-presto-emr/packages/ .
+tar zxvf apache-hive-1.2.1-bin.tar.gz
```

Comment:
Can we make this code independent of the Hive version? That should make upgrades easier.